sovrin_client/agent/agent.py

```diff
@@ -264,4 +264,4 @@ def isSchemaFound(schema, ranViaScript):
         if ranViaScript:
             exit(1)
         else:
-            raise Exception
\ No newline at end of file
+            raise Exception
```

Comment:
It's good that we added the hard return, but the "raise Exception" is eating at me. We should at least raise a RuntimeError with a string.